Bottom: 6373e4dfcdce6a667ef2991da63bf98e8263cdd9
Top:    f663a7ec71ce630403ba5fe3dc875d7572889a58
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-09 15:55:33 -0500

Enable change hooks from Gitlab


---
diff --git a/docker/buildbot-master/buildbot/master.cfg b/docker/buildbot-master/buildbot/master.cfg
index bc7e7ccd..02056a49 100644
--- a/docker/buildbot-master/buildbot/master.cfg
+++ b/docker/buildbot-master/buildbot/master.cfg
@@ -91,7 +91,12 @@ c['www'] = {
                 ]),
             ],
         ),
+    'change_hook_dialects': {
+        'gitlab': {
+            'secret': os.getenv('GITLAB_WEBHOOK_SECRET')
+        }
     }
+}
 
 ####### PROJECT IDENTITY
